docker/Dockerfile.cross_compile

```diff
@@ -0,0 +1,24 @@
+FROM centos:7.6.1810
+
+# Install requirements
+RUN yum install -y wget tar git make redhat-lsb-core autoconf automake libtool glibc-devel libaio-devel openssl-devel apr-devel lksctp-tools
+
+# Install Java
+RUN yum install -y java-1.8.0-openjdk-devel
+
+# Install aarch64 gcc toolchain
+RUN set -x && \
+  wget https://releases.linaro.org/components/toolchain/binaries/4.9-2016.02/aarch64-linux-gnu/gcc-linaro-4.9-2016.02-x86_64_aarch64-linux-gnu.tar.xz && \
+  tar xvf gcc-linaro-4.9-2016.02-x86_64_aarch64-linux-gnu.tar.xz
+
+# Install Maven
+RUN set -x && \
+  wget http://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo -O /etc/yum.repos.d/epel-apache-maven.repo && \
```

Comment:
Is there a https mirror available?

Reply:
Actually we can drop this completely, Netty has mvnw (Maven wrapper) included in the netty repo.